### ISSUE TITLE
Dash button device closes twice when dash button is pressed once

### ISSUE THREAD

**Issue by kiwikern** [mwittig/pimatic-amazing-dash-button#9]
Amazin-Dash-Button: 0.9.12
Pimatic: 0.9.40

I've added a new dash button that I received just a few day ago from the German Amazon store:
![image](https://user-images.githubusercontent.com/2671139/28933983-7cabad30-787f-11e7-8f88-e8be018b7e62.png)

When I click the dash button, it closes twice and hence triggers my corresponding rule twice.
![dash_button_closed_twice](https://user-images.githubusercontent.com/2671139/28933934-4b62d122-787f-11e7-95f1-05fcd1335442.gif)

This does not happen with my two other dash buttons, that I acquired in the US roughly a year ago.

**Comment by mwittig** [comment]
Maybe the new dash board has a different firmware. I'll try to get mine updated too. It should get updated OTA. 
Last time when I tested it a couple of weeks ago the dash got deactivated after a few attempts (dash is only blinking red when pressed and I need to repeat the registration process). Did you also notice similar?

**Comment by kiwikern** [comment]
It did the red blinking when I deactivated the dash button in the Amazon app, because I wanted to get rid of the push notifications ("Finish your dash button registration and select a product"). I have disabled internet access for the dash button through my router (not 100% sure if that works properly). Probably the app talks to the button directly if within the same network.
When your dash button got deactivated, did your app still list it with unfinished registration process or was it gone?
